python/revng/cli/_commands/graphql_upload.py

```diff
@@ -0,0 +1,65 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import asyncio
+import sys
+from argparse import ArgumentParser
+from typing import List
+
+from aiohttp import TCPConnector
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+
+from revng.cli.commands_registry import Command, CommandsRegistry, Options
+
+
+class GraphQLUploadCommand(Command):
```

Comment:
1. What is this command for? If it's never tested (and used) let's not merge it.
2. The name is misleading, this also performs analyses.

Reply:
1. The files:
   ```
   python/revng/daemon/demo_webpage.py
   python/revng/daemon/templates/base.html
   python/revng/daemon/templates/index.html
   ```
   That were dropped as part of this PR were part of the demo page cremo made way back, they were completely unused expect by me for a specific purpose: they easily allowed loading a binary in daemon and running analyses *without* using revng-ui, which is very handy when testing some things. Since those needed analyses to be hardcoded, I've dropped them but added this command to preserve the functionality (which might be useful to others, not just me)

Reply:
We need to test it. I propose to:
1. Drop `revng daemon-self-test` in favor of this tool plus a `--all-artifacts` switch.
2. Rename this tool into `revng graphql` and see where we go from there.